### DISCUSSION TITLE
Why only use Swagger UI in development?

### DISCUSSION THREAD

**Discussion by VictorioBerra** [Dotnet-Boxed/Templates#1788] [General]
https://github.com/Dotnet-Boxed/Templates/blob/4f4ccbc93f5ce79298797669d4557c53cc3517d0/Source/ApiTemplate/Source/ApiTemplate/Startup.cs#L173-L174

I assumed swagger UI was a good tool for documenting an API and being able to easily share that documentation. Why only have it enabled in development?

**Comment by RehanSaeed** [comment]
Mainly for security. If you end up with an old version of the Swagger UI, it could have vulnerabilities in it.
